app/assets/javascripts/angular/app.coffee

```diff
@@ -3,7 +3,10 @@
     'templates',
     'ngRoute',
     'ngResource',
-    'ui.router'
+    'ngAnimate',
```

Comment:
Are we intending to use angular's built in router or ui-router. For what its worth, ui-router far exceeds angular 1.x built in router in functionality. I would highly recommend using it instead of angular's built in router.